multus/multus.go

```diff
@@ -705,7 +705,24 @@ func cmdDel(args *skel.CmdArgs, exec invoke.Exec, kubeClient *k8s.ClientInfo) er
 	if kubeClient != nil {
```

Comment:
Hi @s1061123 , I'm also hitting this issue.
Note that probably kubeClient  == nil
There is no call to k8s.GetK8sClient(in.Kubeconfig, kubeClient) in cmdDel. Maybe it should be added?

Reply:
Thank you for your comment! Yeah, I need to get kubeClient in cmdDel as well. I changed it.